internal/controllers/user_integrations_controller.go

```diff
@@ -1797,6 +1797,8 @@ func (udc *UserDevicesController) registerSmartcarIntegration(c *fiber.Ctx, logg
 }
 
 func (udc *UserDevicesController) registerDeviceTesla(c *fiber.Ctx, logger *zerolog.Logger, tx *sql.Tx, userDeviceID string, integ *ddgrpc.Integration, ud *models.UserDevice) error {
+	// Flag for which api version should be used
+	apiVersion := c.QueryInt("version", 1)
```

Comment:
It seems a little odd to me to have query parameters and a JSON body.

But I'm not too concerned. My feeling is that we would drop this flag in pretty short order. If we are going this route, can we add the query parameter to the Swagger?